diff --git a/dotfiles/.gitconfig b/dotfiles/.gitconfig
index 394aea2..ab680db 100644
--- a/dotfiles/.gitconfig
+++ b/dotfiles/.gitconfig
@@ -1,6 +1,6 @@
 [user]
-	name = Jonathan Porta
-	email = jonathan@jonathanporta.com
+	name = Austin Brown
+	email = austinbrown34@gmail.com
 
 [core]
   excludesfile = $HOME/.gitignore_global
@@ -53,7 +53,7 @@
 	process = git-lfs filter-process
 
 [github]
-	user = jonathanporta
+	user = austinbrown
 
 [gpg]
 	program = gpg2
diff --git a/dotfiles/.zshrc b/dotfiles/.zshrc
index 766e792..39da193 100644
--- a/dotfiles/.zshrc
+++ b/dotfiles/.zshrc
@@ -1,5 +1,5 @@
 # Path to your oh-my-zsh installation.
-export ZSH=/home/portaj/.oh-my-zsh
+export ZSH=$HOME/.oh-my-zsh
 
 # Set name of the theme to load. Optionally, if you set this to "random"
 # it'll load a random theme each time that oh-my-zsh is loaded.
diff --git a/init.sh b/init.sh
index 183b1cf..55d498b 100755
--- a/init.sh
+++ b/init.sh
@@ -1,18 +1,19 @@
 #!/bin/bash
 set -e
 # Make my typical devel directory
-export DEVEL="$HOME/devel/portaj"
+export DEVEL="$HOME/devel/$USER"
 export DOTFILES_CHECKOUT="$DEVEL/dotfiles"
 mkdir -p $DEVEL
 
 # if we already have the dotfiles repo cloned then lets fetch and nothing else. yolo
-if [ -d "$DOTFILES_CHECKOUT" ]; then
-  cd $DOTFILES_CHECKOUT
-  git fetch
+#if [ -d "$DOTFILES_CHECKOUT" ]; then
+#  cd $DOTFILES_CHECKOUT
+#  git fetch
 # if we don't have the repo, let's clone it
-else
-  git clone git@github.com:JonathanPorta/dotfiles.git $DOTFILES_CHECKOUT
-fi
+#else
+#  git clone https://github.com/JonathanPorta/dotfiles.git $DOTFILES_CHECKOUT
+#  https://github.com/JonathanPorta/dotfiles.git
+#fi
 
 # if there is no dotfiles directory under $HOME - link to the dotfiles subfolder in our repo
 if [ ! -d "$HOME/dotfiles" ]; then
diff --git a/run.sh b/run.sh
index d38eaba..18158ff 100755
--- a/run.sh
+++ b/run.sh
@@ -4,12 +4,12 @@ command -v curl >/dev/null 2>&1 || { echo "I require curl but it's not installed
 command -v jq >/dev/null 2>&1 || { echo "I require jq but it's not installed.  Aborting." >&2; exit 1; }
 
 # Make my typical devel directory
-export DEVEL="$HOME/devel/portaj"
+export DEVEL="$HOME/devel/$USER"
 export DOTFILES_CHECKOUT="$DEVEL/dotfiles"
 mkdir -p $DEVEL
 
 # we want to be able to run this and update the authorized keys with whatever we have on GH
-curl https://api.github.com/users/jonathanporta/keys | jq -r '.[] | .key' > $HOME/.ssh/authorized_keys
+curl https://api.github.com/users/austinbrown/keys | jq -r '.[] | .key' > $HOME/.ssh/authorized_keys
 # echo "Updated '$HOME/.ssh/authorized_keys' to:"
 # cat $HOME/.ssh/authorized_keys
 echo "Truncated and wrote $(cat $HOME/.ssh/authorized_keys | wc -l) keys to '$HOME/.ssh/authorized_keys'."
